frontend/style.less

```diff
@@ -0,0 +1,389 @@
+@color_1: #747474;
```

Comment:
ah — interesting, I think the CSS got un-compiled into LESS by some add-on I had... Definitely wouldn't have named variables this poorly, I swear! :)

Feel free to roll back to the old CSS!